redhat/ubi9-openjdk-17-runtime.yaml

```diff
@@ -0,0 +1,9 @@
+osbs:
+  configuration:
+    container:
+      platforms:
+        only:
+        - x86_64
```

Comment:
Are runtime images a `x86_64` only feature? if so, why?

Reply:
No, but at present the internal Compose URIs I have access to for building RHEL9 images are only populated for X86_64, so without it, OSBS attempts multiarch and fails. I am actively trying to find a better internal build process that would not require this.